rails.yml

```diff
@@ -18,3 +18,8 @@ Metrics/BlockLength:
 Metrics/MethodLength:
   Exclude:
     - "db/**/*.rb"
+
+## Rails
+Rails/HttpPositionalArguments:
```

Comment:
was there anything else broken or do other cops pass our standards?

Reply:
Did not notice/remember any other rails-specific cops problems, seems fine.